saleor/graphql/product/sorters.py

```diff
@@ -145,6 +145,7 @@ class ProductOrderField(graphene.Enum):
     LAST_MODIFIED_AT = ["updated_at", "name", "slug"]
     COLLECTION = ["collectionproduct__sort_order", "pk"]
     RATING = ["rating", "name", "slug"]
+    CREATED = ["created_at", "name", "slug"]
```

Comment:
I would rename it to `CREATED_AT` so that the name suggests it's a date; also it's consistent with `LAST_MODIFIED_AT` and `PUBLISHED_AT`.

Reply:
Agree. Just want to point out, that `product` query field is still named `created`.
```
query {
  product {
    created
  }
}
```

Reply:
Right, we should make it deprecated and add `createdAt` next to it at some point.